lib/article_json/export/apple_news/elements/image.rb

```diff
@@ -0,0 +1,19 @@
+module ArticleJSON
+  module Export
+    module AppleNews
+      module Elements
+        class Image < Base
+          # Headline
+          # @return [Hash]
+          def export
+            {
+              role: "image",
+              URL: @element.source_url,
```

Comment:
Don't we have to upload images to apple news? Or would that step happen afterwards?

Reply:
It will happen afterwards as far as I remember.